src/main.cpp

```diff
@@ -133,14 +142,18 @@ int main() {
 	}
 }
 
-void handle_accessories() {
-	if (duration_cast<milliseconds>(timerAccessories.elapsed_time()).count() > ACCESSORIES_TRANSMIT_INTERVAL) {
+void handle_accessories()
+{
+	if (duration_cast<milliseconds>(timerAccessories.elapsed_time()).count() > ACCESSORIES_TRANSMIT_INTERVAL)
+	{
```

Comment:
New-lining opening braces, or impulsively pressing opt-shift-f now that you mostly code in racket?